openhands/server/routes/settings.py

```diff
@@ -55,6 +55,96 @@ async def load_settings(request: Request) -> GETSettingsModel | JSONResponse:
         )
 
 
+@app.get('/settings-custom-secrets', response_model=GETSettingsCustomSecrets)
```

Comment:
@openhands change these routes to GET /secrets, POST /secrets, and DELETE /secrets/{id}

Also fix the lint errors